source/jormungandr/jormungandr/scenarios/ridesharing/ouestgo.py

```diff
@@ -0,0 +1,245 @@
+                driver_alias = offer.get('journeys', {}).get('driver', {}).get('alias')
+                driver_grade = None
+                driver_image = offer.get('journeys', {}).get('driver', {}).get('image')
+                res.driver = rsj.Individual(
+                    alias=driver_alias,
+                    gender=gender_map.get(driver_gender, rsj.Gender.UNKNOWN),
+                    image=driver_image,
+                    rate=driver_grade,
+                    rate_count=None,
+                )
+
+                ridesharing_journeys.append(res)
+
+        return ridesharing_journeys
+
+    def _request_journeys(self, from_coord, to_coord, period_extremity, instance_params, limit=None):
+        """
+
+        :param from_coord: lat,lon ex: '48.109377,-1.682103'
+        :param to_coord: lat,lon ex: '48.020335,-1.743929'
+        :param period_extremity: a tuple of [timestamp(utc), clockwise]
+        :param limit: optional
+        :return:
+        """
+        dep_lat, dep_lon = from_coord.split(',')
+        arr_lat, arr_lon = to_coord.split(',')
+
+        params = {
+            'apikey': self.api_key,
+            'p[passenger][state]': self.passenger_state,
+            'p[driver][state]': self.driver_state,
+            'p[from][latitude]': dep_lat,
+            'p[from][longitude]': dep_lon,
+            'p[to][latitude]': arr_lat,
+            'p[to][longitude]': arr_lon,
+            'signature': 'toto',
+            'timestamp': period_extremity.datetime,
+        }
+
+        headers = {'Authentication': 'key={}'.format(self.api_key)}
+
+        resp = self._call_service(params=params, headers=headers)
+
+        if not resp or resp.status_code != 200:
+            logging.getLogger(__name__).error(
+                'Ouestgo API service unavailable, impossible to query : %s',
+                resp.url,
+                extra={'ridesharing_service_id': self._get_rs_id(), 'status_code': resp.status_code},
+            )
+            raise RidesharingServiceError('non 200 response', resp.status_code, resp.reason, resp.text)
+
+        if resp:
+            r = self._make_response(
+                resp.json(), period_extremity.datetime, from_coord, to_coord, instance_params
+            )
+            self.record_additional_info('Received ridesharing offers', nb_ridesharing_offers=len(r))
+            logging.getLogger('stat.ridesharing.ouestgo').info(
+                'Received ridesharing offers : %s',
+                len(r),
+                extra={'ridesharing_service_id': self._get_rs_id(), 'nb_ridesharing_offers': len(r)},
+            )
+            return r
+        self.record_additional_info('Received ridesharing offers', nb_ridesharing_offers=0)
+        logging.getLogger('stat.ridesharing.Ouestgo').info(
+            'Received ridesharing offers : 0',
+            extra={'ridesharing_service_id': self._get_rs_id(), 'nb_ridesharing_offers': 0},
+        )
+        return []
```

Comment:
This code seems to [never be executed](https://sonarcloud.io/project/issues?resolved=false&types=BUG&pullRequest=3857&id=Hove_navitia&open=AYSfEWh2472y-je6cpe-). Is that on purpose ?